Dockerfile

```diff
@@ -17,37 +17,48 @@ WORKDIR /build
 # as long as your Package.swift/Package.resolved
 # files do not change.
 COPY ./Package.* ./
-RUN swift package resolve
+RUN swift package resolve --skip-update \
+        $([ -f ./Package.resolved ] && echo "--force-resolved-versions" || true)
 
 # Copy entire repo into container
 COPY . .
 
-# Build everything, with optimizations
-RUN swift build -c release --static-swift-stdlib --product Penny
+# Build everything, with optimizations, with static linking, and using jemalloc
+# N.B.: The static version of jemalloc is incompatible with the static Swift runtime.
+RUN swift build -c release \
+      --target Penny \
+      --static-swift-stdlib \
+      -Xlinker -ljemalloc
 
 # Switch to the staging area
 WORKDIR /staging
 
 # Copy main executable to staging area
 RUN cp "$(swift build --package-path /build -c release --show-bin-path)/Penny" ./
 
+# Copy static swift backtracer binary to staging area
+RUN cp "/usr/libexec/swift/linux/swift-backtrace-static" ./
+
 # Copy resources bundled by SPM to staging area
 RUN find -L "$(swift build --package-path /build -c release --show-bin-path)/" -regex '.*\.resources$' -exec cp -Ra {} ./ \;
 
-# Move /Templates/Penny directory to staging area
-RUN mkdir -p ./Templates && mv /build/Templates/Penny ./Templates/Penny && chmod -R a-w ./Templates/Penny;
-
 # ================================
 # Run image
 # ================================
-FROM ubuntu:jammy
+FROM ubuntu:latest
```

Comment:
This is not safe. Swift does not officially support post-Jammy Ubuntu yet.

Reply:
Hmm i believe i only copied it from the new template Dockerfile... i was thinking the same too about Swift support 🤔

Reply:
reverted